npm/ng-packs/packages/oauth/src/lib/strategies/auth-password-flow-strategy.ts

```diff
@@ -40,13 +36,23 @@ export class AuthPasswordFlowStrategy extends AuthFlowStrategy {
   }
 
   async init() {
-    if (!getCookieValueByName(this.cookieKey) && localStorage.getItem(this.storageKey)) {
-      this.oAuthService.logOut();
-    }
+    this.checkRememberMeOption(this.localStorageService);
 
     return super.init().then(() => this.listenToTokenExpiration());
   }
 
+  private checkRememberMeOption(localStorageService: AbpLocalStorageService) {
+    const accessToken = this.oAuthService.getAccessToken();
+    const expireDate = this.oAuthService.getAccessTokenExpiration();
+    const currentDate = new Date().getTime();
+    const rememberMe = getRememberMe(localStorageService);
```

Comment:
Convert to Boolean type
### Example

`Boolean(getRememberMe(service))` //What will be return ?